docs/bosh/README.md

```diff
@@ -398,13 +451,41 @@ Now you have the infrastructure ready to deploy a BOSH director.
 terminal multiplexer such as `tmux` or `screen`.
 
    ```
-   bosh-init deploy manifest.yml
+   bosh2 create-env manifest.yml
    ```
 
-1. Target your BOSH environment:
+1. Use `vim` or `nano` to create a certificate named `ca_cert.pem`:
+
+    ```
```

Comment:
Got it! I didn't realize the ca_cert wasn't being consumed in the manifest. 

Can you jam it in the manifest somewhere and read it that way? If not then I suppose leave it as is.

I totally agree the ops files is the way to go but that should probably coincide with an update to using the (bosh-deployment)[https://github.com/cloudfoundry/bosh-deployment] repo